### PR TITLE
[8.0][stock_quant_cost_segmentation] Add 8.0 api-decorated method

### PR DESCRIPTION
**Pull request by osvalr**
# Changes made:
- Add 8.0 based method to be able to call the `initializing_quant_segmentation` method that is currently defined _only_ for 7.0 api

### CODE REVIEW

**Comment by osvalr** [comment]
@nhomar I've made changes here in order to be discussed directly to the module `stock_quant_cost_segmentation`

**Comment by hbto** [comment]
@osvalr 

Regarding this method
I think that that method can be migrated completely to v8
without the need to use the v7 counterpart.

Regards.

**Comment by osvalr** [comment]
@hbto the change you suggest it have been taken place, (I've squashed into a single commit)

**Comment by osvalr** [comment]
@hbto help me to review that

**Comment by hbto** [comment]
Checking @osvalr

File: stock_quant_cost_segmentation/model/stock.py [hunk 1]
```diff
@@ -134,11 +134,8 @@ def _compute_segmentation(self):
               "`material cost`, `subcontracting cost`, `landed cost` "
               "& `production cost`"))
 
-    @api.v7
-    def initializing_quant_segmentation(
-            self, cr, uid, ids=None, context=None):
-
-        cr.execute('''
+    def initializing_quant_segmentation(self):
```
Comment (hbto) [inline stock_quant_cost_segmentation/model/stock.py:137]: @osvalr 
Line: 
https://github.com/Vauxoo/addons-vauxoo/blob/8.0/stock_quant_cost_segmentation/data/data.xml#L21

Need to be changed to new api signature:
`env['stock.quant'].initializing_quant_segmentation()`

**Comment by hbto** [comment]
@osvalr 
Already check and fixed missing change

waiting for runbot and raditz test

Regards

**Comment by osvalr** [comment]
thanks @hbto

**Pull request merged**